### DISCUSSION TITLE
Questions regarding fraction of drug recycled and passive transport of drug

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1598] [Questions & Problems]
I had two questions regarding how PK-Sim (MoBi) defines some of their parameters and formulas.

1. When looking at the fraction of drug recycled to the plasma relative to the interstitial space, PK sim has FR = 1. However, I believe literature states the FR to be 0.715 (Chang HY, Wu S, Meno-Tetang G, Shah DK. A translational platform PBPK model for antibody disposition in the brain. J Pharmacokinet Pharmacodyn. 2019 Aug 15;46(4):319–38. ) I was wondering why PK-Sim uses FR = 1.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/5e773c20-1c4d-4601-a2ce-74726b3c2610)

2. When looking at the passive transport system for NetMassTransfer_EndosomalToPlasma_FcRn_Complex, we can see that the source of the volume used in the calculation is the same as the the source of the concentration (ie. it uses concentration of molecule in the endosome multiplied by volume of the endosome to calculated the amount of FcRn complex present).
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/bde7c811-fdae-4c82-8e81-ad141d7fafab)

However, when we look at NetMassTransfer_PlasmaToEndosomal, it uses the concentration of the source multiplied by the volume of the target, not the volume of the source (ie. C_pls* V_end, instead of C_pls*V_pls). In order to calculate the amount of drug present in the plasma, shouldn't the formula use the volume of the source container?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/58ede9a1-4a0b-4d74-bb5b-f484861dc03f)

**Comment by wilbertdew** [comment]
Hi @akim2000, 

To start with your last question, this is actually correct, but it might appear initially to be a mistake, here is the reason:
the uptake rate constant (kup) that is used for unbound drug and endogenous IgG is not the same as kup for FcRn and FcRn bound molecules: their ratio is given by the V_pls/V_end ratio. This makes sense, as a smaller fraction of the circulating unbound drug is captured in comparison to the fraction of membrane-bound FcRn when an endosome is formed from the membrane. However, the same rate constant K_uptake is used in PK-Sim for both processes.

For the unbound drug equation kup is K_uptake * Vend/Vpls. If you multiply this with Vpls as you suggest and is the correct thing to do, the equation can be simplified to K_uptake * Vend, as used in PK-Sim. 
For the FcRn molecule kup is the same as K_uptake, so K_uptake * Vpls would be appropriate there, but for some reason I dont fully understand, PK-Sim uses  f_mem_pls * K_uptake *  V_end = Vend/Vpls * K_uptake * Vend = Vpls/Vend * K_uptake * Vend/Vpls * Vpls = K_uptake * Vpls 

With respect to the reason of chosing a FR of 0.75 vs 1, I believe PK-Sim uses 1 for reasons of identifiability, and a clearer separation of the two-pore distribution model and the FcRn-mediated salvaging from clearance model.
Most of this is described in the original publication of this model: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5845054/
Part of the additions above is also described in our recent publication: https://pubmed.ncbi.nlm.nih.gov/36877385/
Hopefully this helps!
Best,
Wilbert

**Comment by Christoph27** [comment]
Hi, 

To your 1. question: In different published PBPK models, different values for this fraction are used. PK-Sim is not the only model using recycling to plasma only. See the paper  https://doi.org/10.1007/s10928-017-9559-4 for different uptake/recycling fractions used in PBPK models and respective citations. At the end, the values are not well identifiable using the usual data for fitting. With the value used in PK-Sim, there is no transcytosis via the endosome, i.e. the two-pore mechanisms is the only mechanism responsible for extravasation. The current parameterisation was chosen to clearly separate the mechanism for extravasation described by the two-pore equation and the endosomal clearance/FcRn mediated recycling. Structurally it is possible to use different fractions for the case there are better data allowing to quantify them.

To you 2. question: 
The uptake mechanism into the endosome is pinocytosis, i.e. a certain fluid volume is endocytosed into the cell and the same fluid volume is going back (to e.g. plasma, see above) to maintain cell volume. The equations in PK-Sim are formulated in a way to describe this fluid volumes as a fraction of endosomal volume (fraction being implicitly present in rate constants). 

Hope that helps.
